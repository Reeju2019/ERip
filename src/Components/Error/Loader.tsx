import './Loader.css'
const Loader: React.FunctionComponent = () => {
  return (
    <>
      <div className='loading'>
        <div></div>
      </div>
    </>
  )
}

export default Loader
