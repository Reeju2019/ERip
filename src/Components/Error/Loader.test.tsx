import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import Loader from './Loader'

test('loader', () => {
  render(<Loader />)
})

describe('loader rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<Loader />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<Loader />)
    ReactDOM.unmountComponentAtNode(div)
  })
})