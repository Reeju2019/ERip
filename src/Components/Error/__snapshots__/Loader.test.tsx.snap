// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader rendering specification SnapShot testing 1`] = `
<div
  className="loading"
>
  <div />
</div>
`;
